SingularityService/src/main/java/com/hubspot/singularity/resources/AgentResource.java

```diff
@@ -173,6 +173,13 @@ public void decommissionAgent(
     );
   }
 
+  @GET
+  @Path("/agent/max-decommissioning-count")
```

Comment:
This isn't really agent specific, and unfortunately doesn't fit neatly with any of our other resources either. Probably worth creating a separate "config" resource class to expose this.

Reply:
haha I was considering doing that but then thought a new resource would be overkill -- I'll make a new one